packages/protocol/contracts/team/airdrop/ERC20Airdrop.sol

```diff
@@ -53,7 +47,25 @@ contract ERC20Airdrop is MerkleClaimable {
         // Check if this can be claimed
         _verifyClaim(abi.encode(user, amount), proof);
 
-        // Transfer the tokens
-        IERC20(token).safeTransferFrom(vault, user, amount);
+        // Transfer the tokens from contract
+        IERC20(token).transfer(user, amount);
+    }
+
+    /// @notice Withdraw ERC20 tokens from the Vault
+    /// @param _token The ERC20 token address to withdraw
+    /// @dev Only the owner can execute this function
+    function withdrawERC20(IERC20 _token) external onlyOwner {
+        // If token address is address(0), use token
+        if (address(_token) == address(0)) {
+            _token = token;
+        }
+        // Transfer the tokens to owner
+        token.transfer(owner(), token.balanceOf(address(this)));
+    }
+
+    /// @notice Withdraw ETH from the Vault
+    /// @dev Only the owner can execute this function
+    function withdrawETH() external onlyOwner {
+        payable(owner()).transfer(address(this).balance);
```

Comment:
i think this one is not needed as there is no r`eceive()`, `fallback()` and any `payable` functions in this contract, so basically noone could send `ETH`.